packages/react-dom/src/events/ChangeEventPlugin.js

```diff
@@ -235,10 +236,7 @@ function handleControlledInputBlur(inst, node) {
   }
 
   // If controlled, assign the value attribute to the current value on blur
```

Comment:
Comment misleading now?

Reply:
I don't think so. Though the controlled input guard triggers above this line. It is not apart of this specific block.

Reply:
But you asked! So we could probably improve it. Something like:

```
// Focused number inputs do not update their defaultValue on change. This
// avoids unexpected value changes in Chrome while the user is typing. On blur,
// we need to synchronize the defaultValue to match standard input behavior.
```